<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementWidget</class>
 <widget class="QWidget" name="ElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="FilterWidget" name="filter" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../editor.qrc">
         <normaloff>:/icons/res/icons/save.png</normaloff>:/icons/res/icons/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveAllButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../editor.qrc">
         <normaloff>:/icons/res/icons/parentdir-128.png</normaloff>:/icons/res/icons/parentdir-128.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+S</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="elementView">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>224</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>224</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTreeView {
     show-decoration-selected: 0;
 }

QTreeView::item:selected {
	background-color: rgb(0, 85, 255);
	 border-radius: 0px;
     border: 0px solid rgb(224, 224, 224);
     padding: 0px;
	color: white;
 }

/*
QTreeView::item:focus {
	background-color: rgb(255, 255, 90);
     border: 1px solid rgb(0, 0, 0);
	 border-radius: 5px;
     padding: -1px;
	 color: black;
 }
*/

 QTreeView::item:hover:!focus {
	 background-color: rgb(85, 170, 255);
	 border-radius: 0px;
     border:0px solid rgb(0, 0, 0);
     padding: 0px;
 }
</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tag</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterWidget</class>
   <extends>QWidget</extends>
   <header>../src/widgets/FilterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ElementWidget</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAllButton</sender>
   <signal>clicked()</signal>
   <receiver>ElementWidget</receiver>
   <slot>onSaveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSave()</slot>
  <slot>onSaveAll()</slot>
 </slots>
</ui>
